README.md

```diff
@@ -1,8 +1,5 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
+# Description
```

Comment:
- [ ] Kindly add the name of your project before description like below to make your readme more descriptive 😊

```
# Vet Clinic
# Description
```